packages/transaction-controller/src/TransactionController.ts

```diff
@@ -3614,6 +3627,73 @@ export class TransactionController extends BaseController<
     return transactionMeta;
   }
 
+  async #updateFirstInteraction(
```

Comment:
I don't think we should do anything now, but if these API usages grow, we could standardise this pattern to reduce some duplication.

Reply:
That's right, I've noticed that as well, it's identical to simulation update.